src/main/java/uk/gov/justice/hmpps/prison/api/resource/CourtResource.kt

```diff
@@ -51,4 +51,33 @@ class CourtResource(private val courtService: CourtService) {
   ): CourtEventDetails? {
     return this.courtService.getNextCourtEvent(bookingId)
   }
+
+  @ApiResponses(
+    ApiResponse(responseCode = "200", description = "OK"),
+    ApiResponse(
+      responseCode = "400",
+      description = "Invalid request.",
+      content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+    ),
+    ApiResponse(
+      responseCode = "404",
+      description = "Requested resource not found.",
+      content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+    ),
+    ApiResponse(
+      responseCode = "500",
+      description = "Unrecoverable error occurred whilst processing request.",
+      content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+    ),
+  )
+  @Operation(summary = "Returns the number of active court cases for the passed in bookingId")
+  @PreAuthorize("hasAnyRole('GLOBAL_SEARCH', 'VIEW_PRISONER_DATA', 'RELEASE_DATES_CALCULATOR')")
+  @GetMapping("/{bookingId}/count-active-cases")
```

Comment:
feel free to just have `RELEASE_DATES_CALCULATOR`  if that is all that needs this endpoint